LoopTests/Managers/Alerts/AlertManagerTests.swift

```diff
@@ -94,77 +94,77 @@ class DeviceAlertManagerTests: XCTestCase {
-        deviceAlertManager.removeAlertResponder(managerIdentifier: DeviceAlertManagerTests.mockManagerIdentifier)
-        deviceAlertManager.acknowledgeDeviceAlert(identifier: Self.mockIdentifier)
+        responder.acknowledged[AlertManagerTests.mockTypeIdentifier] = false
+        alertManager.removeAlertResponder(managerIdentifier: AlertManagerTests.mockManagerIdentifier)
+        alertManager.acknowledgeAlert(identifier: Self.mockIdentifier)
         XCTAssert(responder.acknowledged[Self.mockTypeIdentifier] == false)
     }
     
     func testAcknowledgedAlertsRemovedFromUserNotificationCenter() {
-        deviceAlertManager.acknowledgeDeviceAlert(identifier: Self.mockIdentifier)
+        alertManager.acknowledgeAlert(identifier: Self.mockIdentifier)
     }
     
     func testSoundVendorInitialization() {
         let soundVendor = MockSoundVendor()
-        deviceAlertManager.addAlertSoundVendor(managerIdentifier: Self.mockManagerIdentifier, soundVendor: soundVendor)
+        alertManager.addAlertSoundVendor(managerIdentifier: Self.mockManagerIdentifier, soundVendor: soundVendor)
         XCTAssertEqual("Sounds", mockFileManager.createdDirURL?.lastPathComponent)
         XCTAssertEqual(["\(Self.mockManagerIdentifier)-existsOlder"], mockFileManager.removedURLs.map { $0.lastPathComponent })
         XCTAssertEqual(["doesntExist", "existsOlder"], mockFileManager.copiedSrcURLs.map { $0.lastPathComponent })
         XCTAssertEqual(["\(Self.mockManagerIdentifier)-doesntExist", "\(Self.mockManagerIdentifier)-existsOlder"], mockFileManager.copiedDstURLs.map { $0.lastPathComponent })
     }
     
     // Unfortunately, it is not very easy to test playback of delivered notifications, because we
-    // can't construct UNNotifications.  Hopefully, the code footprint in `DeviceAlertManager.playbackDeliveredNotification` is small enough, because it calls common code under test.
+    // can't construct UNNotifications.  Hopefully, the code footprint in `AlertManager.playbackDeliveredNotification` is small enough, because it calls common code under test.
     
     func testPlaybackPendingImmediateNotification() {
         let date = Date()
-        let content = DeviceAlert.Content(title: "title", body: "body", acknowledgeActionButtonLabel: "label")
-        let alert = DeviceAlert(identifier: Self.mockIdentifier,
+        let content = Alert.Content(title: "title", body: "body", acknowledgeActionButtonLabel: "label")
+        let alert = Alert(identifier: Self.mockIdentifier,
                                 foregroundContent: content, backgroundContent: content, trigger: .immediate)
 
         mockUserNotificationCenter.pendingRequests = [ try! UNNotificationRequest(from: alert, timestamp: date) ]
-        deviceAlertManager = DeviceAlertManager(rootViewController: UIViewController(),
+        alertManager = AlertManager(rootViewController: UIViewController(),
                                                 handlers: [mockPresenter],
```

Comment:
[nit] Just noticed after reviewing a bunch of these files that the indentation is now off on all of the updated code.